code/modules/mob/living/carbon/alien/diona/diona_nymph.dm

```diff
@@ -103,8 +113,7 @@
 		holder_type = null
 
 	species = all_species[new_species]
-	if(species.language)
-		add_language(species.default_language)
+	add_language(species.default_language)
```

Comment:
Shouldn't this be rewritten to be this?
```
if (species.default_language)
    add_language(species.default_language)
```

It seems to exist as a simple safety catch, assuming that everyone has it defined is bad.